### DISCUSSION TITLE
Autofill Suggested Tags in Onboarding With Supported Tags If Suggested Tags Not Set

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16258] [Feature Requests]
**Problem**

Currently, admin manipulate suggested tags from `/admin/customization/config` > Onboarding by manually inputting them into a box. (see image below)

![Screen Shot 2022-01-21 at 2 48 34 PM](https://user-images.githubusercontent.com/16007075/150591144-b1806fe5-2169-467a-a9a1-56ed9059b912.png)

The issue that we somewhat often run into is that admin forget to update this box. 

**Potential Solutions**

I think it would be helpful if suggested tags were not filled, then we could would auto-fill the suggested tags area with any supported tags that have been set up.

If there weren't any supported tags set, then perhaps the section just shouldn't appear at all. 

**Additional Context**

Making it so that the section wouldn't appear at all would also go well with a feature request that @ellativity plans to write in which we let Creators omit certain sections (like suggested users or suggested tags) from onboarding, while also potentially adding other sections (like suggested organizations).

**Comment by ellativity** [comment]
https://github.com/forem/forem/issues/16227

**Comment by Nathanael8426** [comment]
You could also set it as manually, we are also making it custom for different campaigns and post so that we could effectively convert our [leads into customers](https://blog.leadstal.com/convert-your-leads-into-customers/). Try to learn to handle it manually. As auto get mixed up with other tags also. Which sometime effect the post and product which results in less leads.
